common/app/views/fragments/footer.scala.html

```diff
@@ -94,6 +94,9 @@
                     contact us</a></li>
             }
 
+        <li class="colophon__item footer__feedback--hide js-tech-feedback"><a data-link-name="uk : footer : tech feedback">report technical issue</a></li>
```

Comment:
actually I was only adding the switch to remind me to do something about it, I'm not actually going to need to turn it on and off at will